web-ui/src/components/team-member/TeamMemberSelect.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+import Avatar from "@material-ui/core/Avatar";
+
+import "./TeamMemberSelect.css";
+
+const TeamMemberSelect = (props) => {
+  const { teamMembers, onChange, singleSelect = false } = props;
+  const [filteredTeamMembers, setFilteredTeamMembers] = useState(teamMembers);
+
+  const filterTeamMembers = (e) => {
+    let searchInput = e.target.value;
+    let filtered = teamMembers.filter((member) => {
+      return member.name.includes(searchInput);
+    });
+    setFilteredTeamMembers(filtered);
+  };
+
+  const selectMultipleTeamMembers = (member) => {
+    member.selected = !member.selected;
```

Comment:
What is the logic and purpose for this line?

Reply:
I know it looks a bit strange but it's basically a toggle for filtering the selected members and assigning the appropriate className